modules/api/static/schema.yml

```diff
@@ -129,7 +129,7 @@ paths:
             type: string
       responses:
         '200':
-          description: An object containing the list of instrument summary descriptors
+          description: An object containing the list of projects
```

Comment:
This doesn't make sense to me. why is `/projects/{project}/instruments` returning "a list of projects"?

Reply:
ma bad, I was distracted

Reply:
not sure if you fixed it and lost it in the rebase but it still seems to be wrong.

Reply:
@driusan this never happened